### DISCUSSION TITLE
Drone destablizing and crashing when I use go_to() from Position_hl_commander

### DISCUSSION THREAD

**Discussion by bhavya1600** [bitcraze/discussions#440] [Q&A]
Hi,
I need my drone (CF Bolt) to go to a specific coordinate in the Loco Positioning System but am unable to do so using either go_to or even send_setpoints from the commander.  The drone takes off and starts wobbling vigorously in the air before finally crashing. I don't know if I'm doing anything wrong. The drone is stable when controlled using the cf-client. Also, when I print the starting position from pc, no matter where the drone is on the LPS it always returns 0,0,0.
My code is below.

```
def simple_sequence():
    with SyncCrazyflie(uri, cf=Crazyflie(rw_cache='./cache')) as scf:
        with PositionHlCommander(scf, default_velocity=0.1, controller=PositionHlCommander.CONTROLLER_PID) as pc:

            print("Starting Position: "+str(pc.get_position()))
            pc.go_to(2.5, 2.6, 0.4)
            pc.back(0.5)
            pc.land(velocity = default_velocity)

if __name__ == '__main__':
    cflib.crtp.init_drivers()

    simple_sequence()
```

**Comment by knmcguire** [comment]
Hi!

It is strange since this is using the same functionality as the cfclient, namely the High level commander

So the first thing that pops out is that the difference is the default velocity is quite low. The [one of the cfclient](https://github.com/bitcraze/crazyflie-clients-python/blob/473fabd189e74840abb9598df8da3e848c05445a/src/cfclient/ui/tabs/FlightTab.py#L393-L399) has a default velocity of 0.3. Also it sets the default height to 0.3, while yours will probably go to [the standard in the cflib ](https://github.com/bitcraze/crazyflie-lib-python/blob/8d82a769952d1b49de0d6779a2b59ba94036151a/cflib/positioning/position_hl_commander.py#L49-L54)which is 0.5. 

Perhaps try to get it very close to how the cfclient does it first? 

That the pc.get_position is 0,0,0 I wouldn't know and that sounds like a bug, but it is not the true position estimate but the position that the class thinks the crazyflie is based on the commands alone. You can check [these examples](https://github.com/bitcraze/crazyflie-lib-python/tree/master/examples/logging) to setup logging with the [ stateEstimate.x/y/z](https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/api/logs/#stateestimate) variables.
